regions.js

```diff
@@ -0,0 +1,32 @@
+export const REGION_LONDON = 'London'
```

Comment:
Makes perfect sense here for now, just pondering out loud if you think one day these regions and corresponding info might come from the DB.

Reply:
Yes, definitely.